Trustroots/trustroots
# Rename test data to seed data

#### Proposed Changes

* This is a renaming of the test-data scripts to 'seed-data' to better separate 'test' functionality from 'seed' functionality.  

#### Testing Instructions

* Verify the seeding scripts still work as previously with the new names
  * `npm run seed-data`
  * `npm run seed-data:tribes 50`
  * `npm run seed-data:users 50 admin1`
  * `npm run seed-data:messages 50 10` 

* Verify the documentation looks correct.
